qa/tasks/cephfs/test_fragment.py

```diff
@@ -267,7 +267,7 @@ def test_deep_split(self):
 
         def _check_pq_finished():
             num_strays = self.fs.mds_asok(['perf', 'dump', 'mds_cache'])['mds_cache']['num_strays']
-            pq_ops = self.fs.mds_asok(['perf', 'dump', 'purge_queue'])['purge_queue']['pq_executing']
+            pq_ops = self.fs.mds_asok(['perf', 'dump', 'purge_queue'])['purge_queue']['pq_executing_items']
```

Comment:
This is why we should not do this. Please consider that all tests across upgrades will fail.

Reply:
So this came up from a BZ https://bugzilla.redhat.com/show_bug.cgi?id=2314327 where the support and/or the customer were clueless of what this perf counter means since it looks quite similar to `pq_executing_ops`. From my 1x1 with greg last week, we felt the need to make it more distinct and hence this PR.